geometry_msgs/Transform pose0
geometry_msgs/Transform pose1
geometry_msgs/Transform pose2
geometry_msgs/Transform pose3
